moviesite/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Movie Landing Page</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <nav class="nav">
+    <a href="#">Home</a>

```

Comment:
this should link back to /index.html for ese of navigation throught the project, and add your information to the root index.html so that your site can be viewed